source/reference/service-limitations.txt

```diff
@@ -74,7 +74,8 @@ Time Series Collections
 
 :manual:`Time series collections </core/timeseries-collections/>` do not
 yet support :manual:`change streams </changeStreams/>`. Therefore,
-you cannot define triggers on or sync to a time series collection.
+you cannot define triggers on a time series collection. You can sync a time series
+collection, but the collection must be :ref:`asymmetrically synced <stream-data-from-client-to-atlas>`.
```

Comment:
rather than just using 'sync' as verb here, i think it should be explicit that you are syncing using Atlas Device Sync.

smthn like: 

```rst
You can only use a time series collection with Atlas Device Sync
using :ref:`Asymmetric Sync <stream-data-from-client-to-atlas>`.
```

Reply:
Great suggestion! I riffed a bit on your suggestion.
``` rst
Atlas Device Sync supports time series collections through 
:ref:`Asymmetric Sync <stream-data-from-client-to-atlas>`.
```

Reply:
i don't think this is clear enough that you can only use time series collections if you are using Device Sync with Asymmetric Sync. could you rephrase to make it more explicit?

Reply:
@mongodben, yeah, that makes sense. I expanded a bit to make it more explicit.

``` rst
You can use time series collections with Atlas Device Sync, but only 
:ref:`Asymmetric Sync <stream-data-from-client-to-atlas>` is supported. You can't 
use time series collections with Flexible Sync or Partition-Based Sync.
```

Reply:
i'd drop the 2nd sentence. Asymmetric Sync is still using Flexible Sync.

Reply:
Oh, right! It's just one-way. Thanks for the reminder.